addons/web/static/src/views/fields/properties/property_value.xml

```diff
@@ -92,6 +92,7 @@
                     </t>
                     <Many2XAutocomplete
                         t-else=""
+                        t-key="props.comodel + '_' + props.string"
```

Comment:
I usually only see "t-key" set when using a foreach.

How does it work exactly? Is there a hidden mechanism in OWL that forces to component to be reset when its key changes?

Is there a notion of uniqueness involved?

Reply:
It's a bit weird we need it because we already have `resModel="props.comodel"`, so the component should be re-rendered when `props.comodel` change no ?

Reply:
@julien-banken Hi :) 

So if it should be unique, why using string and not name ? (you technically can have 2 fields with the same string on the same label, so the key won't be unique)

Thanks for the explanation :)

Reply:
I added the id of the property field in the key generated to `t-key`. It will ensure that the key will be unique for each component. I still need to use the `comodel` and the `string` props ensure that the component is fully reactive.